src/uikit.cs

```diff
@@ -17043,6 +17586,7 @@ partial interface UIUserNotificationAction : NSCopying, NSMutableCopying, NSSecu
 		NSString ResponseTypedTextKey { get; }
 	}
 #else
+#if !XAMCORE_4_0 // No longer present in watchOS 7.0
```

Comment:
Maybe we should add a `[Deprecated (...)]` then

Reply:
The class itself is already deprecated this being the reason I did not add it.